templates/HAPROXY_HEAD

```diff
@@ -0,0 +1,18 @@
+global
+  daemon
+  log /dev/log local0
+  log /dev/log local1 notice
+  maxconn 4096
+  tune.ssl.default-dh-param 2048
+defaults
+  log               global
+  retries           3
+  maxconn           2000
+  timeout connect   5s
+  timeout client    50s
+  timeout server    50s
+listen stats
+  bind 0.0.0.0:9090
```

Comment:
Deployed to a public slave, this will make the stats accessible to the public internet.
But it's a good point, not sure how to properly solve that. What I actually want here is to map this port to the hosts *internal* interface but I don't think this is possible with marathon.